packages/A11yContent/A11yContent.jsx

```diff
@@ -22,9 +23,9 @@ const A11yContent = ({ children, ...rest }) => {
 
 A11yContent.propTypes = {
   /**
-   * Accessible content.
+   * Accessible content. Can be either text or `<Heading />`  Component
```

Comment:
Can we change this to

```suggestion
   * Accessible content. Can be either a string or a `Heading`  Component
```